list_of_standing_committees_and_working_groups.md

```diff
@@ -0,0 +1,30 @@
+# List of Standing Committees and Working Groups
+
+Project work that expands beyond software-related work is organized into chartered domains and is carried out by Standing Committees and Working Groups. A description of these bodies and their function can be found [here](standing_committees_and_working_groups.md). This document contains a list of the current Standing Committees and Working Groups with charter summaries and links to their GitHub repositories.
+
+## Standing Committees
+
+### Diversity, Equity, and Inclusion
+
+Charter Summary: Increase diversity, equity and inclusion within all aspects of Project Jupyter, including the various councils/leadership, Subprojects, and the broader community.
+
+### Code of Conduct Incident Response
+
+Charter Summary: Own and manage the Jupyter Code of Conduct reporting and incident response processes and make determinations on individual incidents.
+
+### Conflict of Interest
+
+Charter Summary: Ensure that the joint and conflicting interests of leaders across Project Jupyter are disclosed appropriately, handled with transparency, and mitigated with integrity.
+
+## Working Groups
+
+### Trademark and Branding
+
+Charter Summary: License, protect, and promote the trademarks and visual and textual branding of Jupyter.
+
+## Community Building and Events
+
+[Charter](communitybuildingcommittee.md) Summary: Grow, build, and connect the global Jupyter community of users and contributors.
+
+Charter Summary: Build cross-project community, resources, and tools to support the inclusion of disabled people in Jupyter applications and development.
```

Comment:
```suggestion
Charter Summary: Build cross-project community, resources, and tools to support the inclusion of people with disabilities in Jupyter applications and development.
```

@Ruv7 pointed out [this helpful page](https://www.brown.edu/campus-life/support/accessibility-services/resources-teaching-students-disabilities/appropriate-terminology#:~:text=In%20referring%20to%20people%20with,%22differently%20abled%22%20is%20discouraged) so we decided on the Feb 25 governance call to make a minor tweak to this summary.